conference.js

```diff
@@ -2447,36 +2449,34 @@ export default {
 
         APP.UI.removeAllListeners();
 
-        let requestFeedbackPromise;
+        let feedbackResult = true;
 
         if (requestFeedback) {
-            requestFeedbackPromise
-                = APP.store.dispatch(maybeOpenFeedbackDialog(room))
+            try {
+                feedbackResult = await APP.store.dispatch(maybeOpenFeedbackDialog(room, feedbackTitle));
 
-                    // false because the thank you dialog shouldn't be displayed
-                    .catch(() => Promise.resolve(false));
-        } else {
-            requestFeedbackPromise = Promise.resolve(true);
+                if (!feedbackResult.wasDialogShown && feedbackTitle) {
+                    await APP.store.dispatch(openLeaveReasonDialog(feedbackTitle));
```

Comment:
Shouldn't we display the leave reason dialog independent of the request feedback? I mean now the leave reason dialog will be displayed only if the `requestFeeback` is set to `true`, maybe add another parameter to the hangup function `showLeaveReasonDialog`?

Reply:
fair point